.changeset/weak-doors-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@urql/core': minor
```

Comment:
I'd say this is a patch actually, since nothing will have broken in expected usage